src/models/transfers.js

```diff
@@ -435,13 +437,18 @@ function * fulfillTransfer (transferId, fulfillmentUri) {
       }
     } catch (err) {
       // In case of a synchronization error, retry
-      if (Number(err.code) === 40001) {
-        retry(err)
+      if (Number(err.code) === 40001 || err.name === 'InsufficientFundsError') {
```

Comment:
@dappelt yeah, good point, I'll fix https://github.com/interledgerjs/five-bells-ledger/blob/2c5d1abc7112c5fac6e111d2b9a0fd42e8471278/src/lib/holds.js#L10-L13 so that it lets through the original error.

@emschwartz oh wow, seems we're doing two layers of retry, then, [once in five-bells-shared/lib.db.js](https://github.com/interledgerjs/five-bells-shared/blob/master/lib/db.js#L72) using the [retry package](https://www.npmjs.com/package/retry), and [once in five-bells-ledger/src/model/transfers.js](https://github.com/interledgerjs/five-bells-ledger/blob/master/src/models/transfers.js#L381) using the [promise-retry package](https://www.npmjs.com/package/promise-retry). Synchronization error 40001 seems to be specific to IBM DB2. Do you want to